src/DGtal/io/readers/SurfaceMeshReader.h

```diff
@@ -97,6 +98,18 @@ namespace DGtal
     /// created mesh is ok.
     static
     bool readOBJ( std::istream & input, SurfaceMesh & smesh );
+
+    /// Reads an input file as an OBJ file format and outputs the
+    /// corresponding surface mesh.
+    ///
+    /// @param[in,out] input the input stream where the OBJ file is read.
+    /// @param[out] smesh the output surface mesh.
+    /// @param[out] materials a vector containing the material of each face (an index).
```

Comment:
```suggestion
    /// @param[out] materials a vector containing the material of each face (an index, may be empty)
```
If there is no material info, an empty vector is retuned, right ?